test/Libraries/RevitNodesTests/Elements/DocumentTests.cs

```
@@ -17,5 +21,33 @@ public void Current()
             Assert.IsTrue(doc.IsFamilyDocument);
         }
 
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanSaveFamiliesInCurrentDocument()
+        {
+            // Arrange
+            var saveableFamily = ElementSelector.ByElementId(110049, true);
+            var noneditableFamily = ElementSelector.ByElementId(20915, true);
+            string savedFamilyName = saveableFamily.Name + ".rfa";
+
+            string tempFolder = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+            Directory.CreateDirectory(tempFolder);
```

we need another test where the directory doesn't already exist - we should either throw or (preferably) create the missing directory